### ISSUE TITLE
Chrome flickering on samus xenial

### ISSUE THREAD

**Issue by bengolds** [dnschneid/crouton#2958]
```
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20161129162558~master:5f2f11f9
release: xenial
architecture: amd64
xmethod: xorg
targets: touch,extension,unity-desktop,chrome,audio
host: version 8743.85.0 (Official Build) stable-channel samus 
kernel: Linux localhost 3.14.0 #1 SMP PREEMPT Tue Nov 15 18:25:51 PST 2016 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
```
Chrome has major screen flickering on Ubuntu right now. Bug is [here](https://bugs.launchpad.net/ubuntu/+source/xserver-xorg-video-intel/+bug/1586539). There's a workaround in the [comments](https://bugs.launchpad.net/ubuntu/+source/xserver-xorg-video-intel/+bug/1586539/comments/15) but crouton overwrites 20-intel.conf with 20-crouton-intel-sna.conf, which doesn't include the workaround change.

The bug is about 6 months old without a fix, so maybe in the meantime the workaround should be included by default?

**Comment by octopusnz** [comment]
I can replicate this one as well.
There's also a Chromium bug with more details here: 

https://bugs.chromium.org/p/chromium/issues/detail?id=606152#c73

Seems like there are two schools of thought in those threads - 

a) changes to intel-sna.conf file
b) Removing the xserver-xorg-video-intel package entirely

I guess crouton could do either or both depending on what works for the majority of users.

**Comment by octopusnz** [comment]
Some more wisdom here:

https://bugs.launchpad.net/xorg-server/+bug/1586539

You can actually install the new X stack - some more info here: 

https://wiki.ubuntu.com/Kernel/LTSEnablementStack

```
 sudo apt-get install --install-recommends linux-generic-hwe-16.04 xserver-xorg-hwe-16.04
```

**Comment by octopusnz** [comment]
So it looks like this has been resolved in the linux kernel - as of 4.10.
I know kernel updates for ChromeOS are not straight forward but it seems like this might be a good candidate to be merged in to a ChromeOS kernel release branch as it affects Chrome, albeit in Linux in a chroot.

See: https://bugs.freedesktop.org/show_bug.cgi?id=94987
Here is the patch in the i915 drivers: https://patchwork.kernel.org/patch/9687145/

@dnschneid I know this isn't strictly crouton related but with your contacts and wisdom - do you have someone you could reach out to, to mention this? I can also log a bug I guess against ChromeOS somewhere, but it would have more gravitas coming from you. My understanding is this will be affecting all distributions and many Chromebooks that have this intel graphics chipset so could be a decent number of crouton users.

There were open bugs against Xorg and Ubuntu that have been closed since it's fixed in the kernel.
Chromium still has an open bug here: https://bugs.chromium.org/p/chromium/issues/detail?id=606152
but they might also close it at some point rather than introduce a work-around. So it's likely that crouton users might be the edge cases that get left without a fix unless a change is made in ChromeOS.

**Comment by octopusnz** [comment]
So I've attempted to log this bug with the Chrome OS folks.
If anyone has any additional information or wisdom please feel free to add it to this thread: https://bugs.chromium.org/p/chromium/issues/detail?id=779341

Basically I reckon we need to back port any fixes to the i915 drivers from recent stable Linux kernels into Chrome OS so that the issue can be resolved.
